.storybook/preview.js

```diff
@@ -34,7 +43,11 @@ export const parameters = {
       { name: 'Gemeente Utrecht', class: 'utrecht-theme', color: '#cc0000' }
     ],
   },
-}
+};
+
+export const decorators = [
+  ConfigDecorator,
+];
```

Comment:
looks like there is a better option after all via https://github.com/storybookjs/storybook/issues/16999#issuecomment-1127503783, one more thing to try

Applying the decorators globally does render the source code preview properly, while doing it at the Meta/Story level breaks in various ways depending on how you use it.